modules/reference/pages/systems/ssm-task-log.adoc

```diff
@@ -1,16 +1,15 @@
-= SSM Task Log
+[[ssm_task_log]]
+= System Set Manager Task Log
 
-The SSM Task Log lists all tasks performed against {productname} servers when using SSM. Click on an task's description to see more details.
+The menu:Systems[System Set Manager > Task Log] section shows all actions applied to clients that you have added to the system set manager (SSM).
```

Comment:
We sometimes capitalize "system set manager" on first occurrence within a section, sometimes not.

Reply:
Yes, I think we're a bit inconsistent here. Probably best to address in a new PR, though.

Reply:
https://github.com/SUSE/spacewalk/issues/12356